### ISSUE TITLE
VPN provider support: PrivateVPN

### ISSUE THREAD

**Issue by Silversurfer79** [qdm12/gluetun#2403]
Hi,

PLease if we could add supprt for tyhis I would be very greatful. 

- Publicly accessible URL to a zip file containing the Openvpn configuration files - https://privatevpn.com/client/PrivateVPN-TUN.zip
- Publicly accessible URL to the list of servers - https://privatevpn.com/why-privatevpn/server-locations

Also see: https://github.com/qdm12/gluetun/discussions/2393

Thanks

**Comment by qdm12** [comment]
PrivateVPN is already supported, although it's using the openvpn zip file.
I changed your issue to change the servers updating mechanism to use the website (82 servers) instead of the zip file (79 servers). This is obviously low priority since the zip file approach still works fine.

**Comment by Silversurfer79** [comment]
> PrivateVPN is already supported, although it's using the openvpn zip file. I changed your issue to change the servers updating mechanism to use the website (82 servers) instead of the zip file (79 servers). This is obviously low priority since the zip file approach still works fine.

Yes it does work, but it doesnt support port forwarding. Is there now way to allow this in a generic format? 

pls see: https://github.com/qdm12/gluetun/discussions/2393

**Comment by qdm12** [comment]
> Yes, it does work, but it doesn’t support port forwarding. Is there no way to allow this in a generic form?

Ha! Lucky you (and me), I need someone to confirm this works: #2285 ! Please report on existing issue #1859 

> In the zip file above, there are 5 locations that support it

Where did you get that list of 5 servers supporting port forwarding? I need this ideally to automate it instead of hardcoding server hostnames compatible with port forwarding. In the meantime, you can set the correct hostname supporting port forwarding for example `-e SERVER_HOSTNAMES=se-sto.pvdata.host` to test the image on #2285 

Marking this and the last few comments about port forwarding as off topic, and let's continue on #1859 about this.
